specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/examples/WafPolicyCreateOrUpdate.json

```diff
@@ -7,6 +7,24 @@
     "parameters": {
       "location": "WestUs",
       "properties": {
+        "policySettings": {
+          "logScrubbing": {
+            "state": "Enabled",
+            "scrubbingRules": [
+              {
+                "state": "Enabled",
+                "matchVariable": "RequestArgNames",
+                "selectorMatchOperator": "Equals",
+                "selector": "test"
+              },
+              {
+                "matchVariable": "RequestIPAddress",
```

Comment:
Is State not needed here?

Reply:
Default value is "Enabled" if it is not specified. Let me add state as well.